
# Week 1 output
## 1. Spades meta running 
> `spades -1 /data/metagenomes/160523Alm_D16-4703_1_sequence.fastq.gz -2 /data/metagenomes/160523Alm_D16-4703_2_sequence.fastq.gz --meta -o spades_final_proj_output -m 16 -t 2`
![Screen%20Shot%202018-11-06%20at%2012.41.44%20AM.png](attachment:Screen%20Shot%202018-11-06%20at%2012.41.44%20AM.png)
As a the above error suggests, there's no ~190G memory for the file to be processed successfully, and so following the instruction, the output log is attached and histogram cannot be plotted successfully.

## 2. Quantify abundance of microbes: upload both fastq files onto the one codex server, run annotation
> `onecodex upload /data/metagenomes/160523Alm_D16-4703_1_sequence.fastq.gz` `onecodex upload /data/metagenomes/160523Alm_D16-4703_2_sequence.fastq.gz`
![Screen%20Shot%202018-11-07%20at%202.50.58%20PM.png](attachment:Screen%20Shot%202018-11-07%20at%202.50.58%20PM.png)
![Screen%20Shot%202018-11-07%20at%202.50.17%20PM.png](attachment:Screen%20Shot%202018-11-07%20at%202.50.17%20PM.png)
![Screen%20Shot%202018-11-07%20at%202.50.02%20PM.png](attachment:Screen%20Shot%202018-11-07%20at%202.50.02%20PM.png)
![Screen%20Shot%202018-11-07%20at%202.52.22%20PM.png](attachment:Screen%20Shot%202018-11-07%20at%202.52.22%20PM.png)
![Screen%20Shot%202018-11-07%20at%203.07.20%20PM.png](attachment:Screen%20Shot%202018-11-07%20at%203.07.20%20PM.png)

## Analysis: What is in the soil
> 
